src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  let copySum;
+
+  const adder = (value) => {
+    if (value) {
+      sum += value;
+
+      return adder;
+    } else {
+      copySum = sum;
+      sum = 0;
+
+      return copySum;
+    }
```

Comment:
```suggestion
    }
    
    copySum = sum;
    sum = 0;

    return copySum;
```
Учитывая что в if есть return - else не нужен